README.md

```diff
@@ -98,4 +128,4 @@ console.log(i)
 ```
 
 -- YOUR ANSWER HERE --
-
+I have no idea. I have never seen a for loop using console.log(). 
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.